.circleci/config.yml

```diff
@@ -149,7 +149,7 @@ jobs:
             set +e
 
             npx cypress run \
-              --env TAGS='not @E2E and not @smoke',USERNAME=${CYPRESS_USERNAME_local},PASSWORD=${CYPRESS_PASSWORD_local},USERNAME_SPO=${CYPRESS_USERNAME_SPO_local},PASSWORD_SPO=${CYPRESS_PASSWORD_SPO_local},USERNAME_ACO=${CYPRESS_USERNAME_ACO_local},PASSWORD_ACO=${CYPRESS_PASSWORD_ACO_local} \
+              --env TAGS='not @E2E and not @smoke',USERNAME=${CYPRESS_USERNAME_local},PASSWORD=${CYPRESS_PASSWORD_local},USERNAME_SPO=${CYPRESS_USERNAME_SPO_local},PASSWORD_SPO=${CYPRESS_PASSWORD_SPO_local},USERNAME_ACO=${CYPRESS_USERNAME_ACO_local},PASSWORD_ACO=${CYPRESS_PASSWORD_ACO_local},USERNAME_PPCS=${CYPRESS_USERNAME_PPCS_local},PASSWORD_PPCS=${CYPRESS_PASSWORD_PPCS_local},MAKE_RECALL_DECISION_API_URL=${CYPRESS_MAKE_RECALL_DECISION_API_URL_local},HMPPS_AUTH_EXTERNAL_URL=${CYPRESS_HMPPS_AUTH_EXTERNAL_URL_local} \
```

Comment:
Just for my understanding: why do the MAKE_RECALL_DECISION_API_URL and HMPPS_AUTH_EXTERNAL_URL values need to be set now (i.e. why weren't they needed for the non-PPCS cases)?

Reply:
This is because this will now make direct calls to the api to "pre create" the recommendation in preparation for the PPCS journey test as a "Given a recommendation exists" test rather than "Given that the entire PO journey is fully functional before we can run this" test.
As such as use the service URL is to make create/update/update status calls directly to the UI. The hmpps auth url is used to retrieve a client access token to authorise these calls.